### PR TITLE
Update and rename CHANGELOG to CHANGELOG.md

### PR DESCRIPTION
**Pull request by EwoutH**
Converts the Changelog to markdown syntax.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1016,11 +1017,11 @@ RELEASE 1.2-beta
 - Fix bug where HTML messages with invalid/excessive css styles couldn't be displayed (#4905)
 - Fix redundant blank lines when using HTML and top posting (#4927)
 - Fix redundant blank lines on start of text after html to text conversion (#4928)
-- Fix HTML sanitizer to skip <!-- node type X --> in output (#4932)
+- Fix HTML sanitizer to skip <! node type X > in output (#4932)
```
Comment (alecpl) [inline CHANGELOG.md:1020]: Changes like these aren't acceptable.

**Review by alecpl — CHANGES_REQUESTED** [review]
Access to the new file will have to be blocked in the .htaccess file.

**Comment by EwoutH** [comment]
@alecpl Thanks for your comments! I did should have used stricter requirements on my regex replace to remove the dashed lines (-----), that is now fixed. I also fixed a few `<!>` cases that got recognized as comments, and replaces a ' that should have been a `.

**Comment by alecpl** [comment]
Please, resolve conflicts and I'll merge it.

**Pull request merged**